app/code/Magento/Checkout/view/frontend/web/template/shipping-address/shipping-method-item.html

```diff
@@ -15,14 +15,17 @@
                attr="'aria-labelledby': 'label_method_' + method.method_code + '_' + method.carrier_code + ' ' + 'label_carrier_' + method.method_code + '_' + method.carrier_code,
                     'checked': element.rates().length == 1 || element.isSelected" />
     </td>
-    <td class="col col-price">
+    <td class="col col-price"
+        attr="'data-table-column': 'label_price_' + method.method_code + '_' + method.carrier_code"
+    >
         <each args="element.getRegion('price')" render="" />
     </td>
     <td class="col col-method"
         attr="'id': 'label_method_' + method.method_code + '_' + method.carrier_code"
         text="method.method_title" />
     <td class="col col-carrier"
-        attr="'id': 'label_carrier_' + method.method_code + '_' + method.carrier_code"
+        attr="'id': 'label_carrier_' + method.method_code + '_' + method.carrier_code,
+             'data-table-column': 'label_carrier_' + method.method_code + '_' + method.carrier_code"
```

Comment:
Why do we need to duplicate it in both TDs?

`<tr data-container-for="shipping-method"` and using existing `ko-value="method.carrier_code + '_' + method.method_code"` should be sufficient.